bin/ruby-build

```diff
@@ -1081,6 +1081,16 @@ has_broken_mac_openssl() {
   [[ "$RUBY_CONFIGURE_OPTS" != *--with-openssl-dir=* ]]
 }
 
+has_openssl_3() {
```

Comment:
I don't think there is strong need to have `has_openssl_3` a separate functions, so let's inline this into `needs_openssl`